visualization/app/codeCharta/util/nodeDecorator.ts

```diff
@@ -99,7 +100,9 @@ export const NodeDecorator = {
 					setNodeMediansToParent(medians, selector, parentSelector, data, name, isDeltaState)
 					collectNodeMediansOnParent(medians, parentSelector, data, name, isDeltaState)
 				} else {
-					parent.data.attributes[name] += data.attributes[name]
+					if (parent.data.attributes[name] !== undefined) {
+						parent.data.attributes[name] += data.attributes[name]
+					}
```

Comment:
I am not sure why this is needed. I would expect that both (the parent and children) have the same set of attributes.
If this case occurs in delta mode, then propably some attributes of a (parent) folder would be missing.

Reply:
Now I understand. Ignore my comment. I was wrong. :-)